http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -374,6 +374,9 @@ private static String stripOauthParametersFromRedirect(String uri) {
 
     private String rewrittenRedirectUri(String originalUri) {
         Map<String, String> rewriteRules = deployment.getRedirectRewriteRules();
+        if (rewriteRules == null || rewriteRules.isEmpty()) {
+            return originalUri;
+        }
```

Comment:
These lines should probably be removed, the rebuilding of the string should be consistent whether a rewrite rule is executed or not.

Reply:
Hi! Yes you have right, but i thougth if no rewriteRule is, why separate and reconstruct the uri. It can't be performancebottleneck, but a little bit slower than this ugly if. If you think after this I should remove my if, after your reply I will.

Thanks

Reply:
Hey, just ran some checks and it doesn't look like it should cause any issues, so I'll close this comment.

Reply:
Thanks a lot!